.cm/platform_labels.cm

```diff
@@ -150,10 +150,13 @@ platforms:
   - ide:
     name: 'ide'
     subprojects:
-      - 'platforms/ide/'
-      - 'subprojects/ide/'
-      - 'subprojects/tooling-api/'
-      - 'subprojects/tooling-api-builders/'
+      - 'platforms/ide/base-ide-plugins'
+      - 'platforms/ide/ide'
+      - 'platforms/ide/ide-native/'
+      - 'platforms/ide/ide-plugins/'
+      - 'platforms/ide/problems/'
```

Comment:
❗ We have this marked as a `Core/runtime` project in the Code Areas sheet, which seems correct to me - this is not merely IDE-related code.  Did this get change?

Reply:
Discussed, this is the temporary home during development.  Please add a comment to tjhe platform in `CODEOWNERS` to note this.